docs/source/index.rst

```diff
@@ -59,6 +59,12 @@ GitHub repo: https://github.com/argonne-lcf/dlio_benchmark.
 
    copyright
    license
+
+.. toctree::
+   :maxdepth: 2
+   :caption: How to install and run DLIO benchmark on Lassen
+
+   instructions_lassen
```

Comment:
You have added this in the end. There is a section called TESTED SYSTEMS AND KNOWN ISSUES -> TESTED SYSTEMS. I would recommend you to split it off as TESTED SYSTEM and KNOWN ISSUES. 

Then within TESTED SYSTEMS, you can add the lassen installation and running instructions there. 

This organization is odd.

Reply:
Fixed. But we have to see the generated output to make sure.